### ISSUE TITLE
Player model for Minecraft skin not showingf

### ISSUE THREAD

**Issue by radstevee** [JannisX11/blockbench#2056]
### What are you trying to do, and what do you expect to happen?

I'm trying to edit a Minecraft skin. I click on minecraft skin => select my file => Confirm, and expecting the player model to render so I can edit the skin

### What happens instead?

The player model doesn't show, it looks like this:
![image](https://github.com/JannisX11/blockbench/assets/96977790/382a2d65-2d28-4cda-b740-613683006233)

### Model format in which the issue occurs

Minecraft skin (.png)

### Blockbench variant

Program

### Blockbench version

4.8.3

### Operating System

GNU/Linux

**Comment by JannisX11** [comment]
Does this also happen in other formats?
Which distro of Linux are you using?

**Comment by radstevee** [comment]
Yes, it does. Archlinux btw (nvidia, X11)On 16/10/2023, 23:47 JannisX11 ***@***.***> wrote:

   
  Does this also happen in other formats? Which distro of Linux are you using?

**Comment by JannisX11** [comment]
Duplicate of #1964
